[14.0][ADD] sale_single_attribute_value_display (#2227)
OCA/sale-workflow: sale_single_attribute_value_display/readme/USAGE.rst

> Install module. Create product with single value attribute. Add it to Sale Order Line.

Reviewer: Put it as numbered bullet list:

```suggestion
#. Create a product with any single value attribute.
#. Add it to a sales order.
#. ...
```
